### ISSUE TITLE
Write an AWS CDK Construct to deploy Komiser

### ISSUE THREAD

**Issue by mlabouardy** [tailwarden/komiser#897]
**Describe the bug**
It would be nice to have a CDK to deploy Komiser as a Helm Chart to EKS to make it easier for users to deploy Komiser

**Comment by sshiv5768** [comment]
Hey, @mlabouardy I would like to work on this issue. Can you assign me this.

**Comment by ShubhamPalriwala** [comment]
All yours @sshiv5768

**Comment by sshiv5768** [comment]
Thanks @ShubhamPalriwala for assigning me.

**Comment by sshiv5768** [comment]
Hey, @ShubhamPalriwala is there any reference for this issue.

**Comment by sshiv5768** [comment]
Thanks @AvineshTripathi. Where should be CDK files placed? @ShubhamPalriwala

**Comment by mlabouardy** [comment]
> Thanks @AvineshTripathi. Where should I place CDK files? @ShubhamPalriwala

You can place them [here](https://github.com/tailwarden/helm) :)

**Comment by jakepage91** [comment]
Hey there @sshiv5768 I was wondering if you had an update for this issue? Thanks a million!

**Comment by sshiv5768** [comment]
Yeah, actually I created a helm chart but minor few changes are remaining, will be completed by the end of this week.

**Comment by jakepage91** [comment]
Ideal! Thanks a million

**Comment by sshiv5768** [comment]
Hey @jakepage91, I have raised a PR for this issue, review once.

**Comment by jakepage91** [comment]
I will review this today! Thanks a million @sshiv5768
